stats/equilibria.php

```diff
@@ -59,8 +41,28 @@ function display_graph($d)
     }
 
     if ($total_pages == 0) {
-        dpgraph_error(_("No pages saved in specified range"), 660, 400);
+        $title = _("No pages saved in specified range");
```

Comment:
I feels weird to have this as the page title. Perhaps have the title be `_("Graph error")` and have an `$error = _("No pages saved in specified range");` that gets output in an `<p class='error'>` after the title if defined?

Reply:
good points. I have implemented where the graph title shows and the error show below, that way the user is aware of which chart has an error.